hw3.1/index.html.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style1.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Homework3</title>
+</head>
+<body>
+    <header>
+            <div>
+                <a href="#"><img class="banner" src="images/title.gif" alt="link"></a>
+            </div>
+
+            <div class="images">
+                <img class="big" src="images/img00.jpg" alt="">
+                <img class="big" src="images/img01.jpg" alt="">
+                <img class="big" src="images/img02.jpg" alt="">
+                <img class="big" src="images/img03.jpg" alt="">
+            </div>
+            <nav class="top">
+                <ul class="top1">
+                <a class="link" href="#"><li>Home</li></a>
+                <a class="link" href="#"><li>About</li></a>
+                <a class="link" href="#"><li>Services</li></a>
+                <a class="link" href="#"><li>Contact</li></a>
+                </ul>
+            </nav>

```

Comment:
Make them as 1 row with 4 links (image and span inside link tag)

Reply:
fix)